### ISSUE TITLE
Conversion of [Model]ForSequenceClassification logits to probabilities

### ISSUE THREAD

**Issue by sgummidipundi** [huggingface/transformers#1858]
## ❓ Questions & Help

I noticed that whenever I would convert logits coming from the model to probabilities using the following equation:

probability = e^logit/(1 + e^logit)

The probabilities for my two classes do not add up to 1. Why is this?

**Comment by loveritsu929** [comment]
Logits are the classification scores before softmax.
I don't see why the results of your equation should add up to 1.
You may try to use the softmax function directly.
